File: src/SOP8/Egovframework/Base/base-account-ibll/src/main/java/egovframework/base/account/ibll/request/SaveOption.java
package egovframework.base.account.ibll.request;

import java.util.ArrayList;
import java.util.List;
import egovframework.base.account.ibll.models.UserOption;

public class SaveOption {

    private int userNo = -1;
    private List<UserOption> options = new ArrayList<>();

    public int getUserNo() {
        return userNo;
    }

    public void setUserNo(int userNo) {
        this.userNo = userNo;
    }

    public List<UserOption> getOptions() {
        return options;
    }

    public void setOptions(List<UserOption> options) {
        this.options = options;
    }
}

File: src/SOP8/Base/Account/Base.Account.IBLL/Request/SaveOption.cs
using System.Collections.Generic;

namespace Base.Account.IBLL.Request
{
    using Models;

    public class SaveOption
    {
        private int m_nUserNo = -1;
        private List<UserOption> m_options = new List<UserOption>();

        public int UserNo
        {
            get { return m_nUserNo; }
            set { m_nUserNo = value; }
        }

        public List<UserOption> Options
        {
            get { return m_options; }
            set { m_options = value; }
        }
    }
}
